tests/Microsoft.Identity.Test.Android.UIAutomation/XamarinDroidTests.cs

```diff
@@ -69,35 +73,18 @@ public void InitializeBeforeTest()
         [Test]
```

Comment:
have you merged the latest changes? these now run under one test and the FB tests should be commented out until AppCenter gets back to us w/the issue :/